src/pages/public/ramos/index.jsx

```diff
@@ -1,6 +1,60 @@
+import lobinho from '../../../../public/lobinho.svg';
+import { SectionTitle } from '../../../components/section-title';
+import flordeliz from '../../../../public/flor-de-liz.svg';
+
 function Ramos() {
   return (
-    <h1>Ramos</h1>
+    <>
```

Comment:
Vai precisar transformar esse `<></>` em uma tag `<section></section>` para que possamos controlar a largura máxima do conteúdo quando a tela for maior do que `1160px`. Não sei se você lembra, mas precisamos colocar algo como `max-w-[1160px]` e outros detalhes como os paddings.